github/copilot.go

```diff
@@ -498,110 +465,6 @@ func (s *CopilotService) GetSeatDetails(ctx context.Context, org, user string) (
 	return seatDetails, resp, nil
 }
 
-// GetOrganizationUsage gets daily breakdown of aggregated usage metrics for Copilot completions and Copilot Chat in the IDE across an organization.
-//
-// GitHub API docs: https://docs.github.com/rest/copilot/copilot-usage#get-a-summary-of-copilot-usage-for-organization-members
-//
-//meta:operation GET /orgs/{org}/copilot/usage
```

Comment:
I'm fine with deleting since the removed code has not been officially released in this repo with any tagged release.

That's why I removed the Breaking Change label.

Reply:
Maybe it's better to remove "BREAKING CHANGE" from the PR's description. This will avoid any confusion in for the future PR's readers.

<img width="941" alt="image" src="https://github.com/user-attachments/assets/bdef6261-b5cc-4579-b681-c74772add385">

Reply:
I updated the description, hopefully it's more clear now.